pkgs/applications/editors/bluej/default.nix

```diff
@@ -49,6 +53,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.bluej.org/";
     sourceProvenance = with sourceTypes; [ binaryBytecode ];
     license = licenses.gpl2ClasspathPlus;
+    mainProgram = pname;
```

Comment:
this is technically not necessary, eg `nix run` works without this change
is there a general consensus on how to deal with changes to mainProgram like this ?

Reply:
`lib.getExe` recently [deprecated the fallback of the binary name to the pname](https://github.com/NixOS/nixpkgs/blob/master/lib/meta.nix#L149). Because I was cleaning the (very similar) definition of greenfoot anyway, I made some similar changes here.

Reply:
great ! thanks for the link :heart: